### ISSUE TITLE
Better Documentation of all functions

### ISSUE THREAD

**Issue by tensarflow** [BogdanovKirill/RtspClientSharp#92]
Thanks for the software looks really nice :)
It is not really clear what this library can do. The examples are not enough. 

Can this library send video streams or is it only for receiving and decoding?

**Comment by rageshS** [comment]
Yes absolutely. The repository author needs more attention to the documentation. Otherwise, it will work like a junk program and not trustworthy to apply the production-ready project.

One more important thing, if you have your own repository, please look at the issues section on a daily basis and reply to the reported issues. If you are not willing to do otherwise this repository becomes a tourist trap.
